libsplinter/src/events/reactor.rs

```diff
@@ -135,15 +136,29 @@ impl Reactor {
         }
     }
 
+    /// Return a ReactorShutdownSignaler, for use to send messages to
```

Comment:
This comment is confusing. Can it be reworded?

Reply:
Updated